ceph-salt-formula/salt/ceph-salt/cephbootstrap.sls

```diff
@@ -21,6 +11,17 @@ osd crush chooseleaf for single-node cluster:
     - makedirs: True
     - failhard: True
 
+{% set global_configs = pillar['ceph-salt'].get('bootstrap_ceph_conf', {}).get('global', {}) %}
+
+create bootstrap ceph conf:
+  cmd.run:
+    - name: |
+        echo -en "[global]\n" > /tmp/bootstrap-ceph.conf
+{% for k, v in global_configs.items() %}
+        echo -en "        {{ k }} = {{ v }}\n" >> /tmp/bootstrap-ceph.conf
+{% endfor %}
```

Comment:
not every item should end up in the `[global]` section. can we simply pass-through the config?

Reply:
I've now changed it to not create the `[global]` section if no global configs are defined.